docs/src/tutorials/conic/introduction.md

```diff
@@ -0,0 +1,45 @@
+# Introduction
+
+[Conic programs](https://en.wikipedia.org/wiki/Conic_optimization) are a class
+of convex nonlinear optimization problems which use cones to represent the
+nonlinearities. They have the form:
+```math
+\begin{align}
+    & \min_{x \in \mathbb{R}^n} & f_0(x) \\
+    & \;\;\text{s.t.} & f_i(x) \in \mathcal{S}_i & \;\; i = 1 \ldots m
```

Comment:
```suggestion
    & \;\;\text{s.t.} & f_j(x) \in \mathcal{S}_j & \;\; j = 1 \ldots m
```

Reply:
I suggest to use `j` for constraint indices and `i` for variable indices to be consistent with the intro of LP and NLP